content/docs/guides/branching-schema-only.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Schema-only branches
+subtitle: Protect sensitive data with schema-only branches
+enableTableOfContents: true
+updatedOn: '2025-01-10T00:37:35.161Z'
+---
+
+<EarlyAccess />
+
+Neon allows you to create branches that replicate only the database schema from a source branch, without copying any of the actual data. This feature is particularly valuable when working with sensitive information. Rather than creating branches that include confidential data, you can duplicate just the database structure and then populate it with your own data. This provides your team with a secure and compliant environment for development and testing using Neon branches.
+
+## Creating schema-only branches
+
+You can create schema-only branches in the Neon Console or using the Neon API, in much the same way you create any Neon branch.
+
+<Tabs labels={["Neon Console", "API"]}>
+
+<TabItem>
+
+To create a schema-only branch from the Neon Console:
+
+1. In the console, select your project.
+2. Select **Branches**.
+3. Click **Create branch** to open the branch creation dialog.
+4. For **Type**, select **Schema-only**.
```

Comment:
Would you consider swapping 4 and 5 round? Only because branch name is the first field, the type dropdown/select is lower down the page. (not essential)

Reply:
Thank you. I've updated these steps to what the revised UI (coming soon) will look like (similar to the one below). I'll check back once the updated UI is on staging.
![image](https://github.com/user-attachments/assets/aab427d8-7d54-4338-b297-39347d9f2635)